config/tags.php

```diff
@@ -314,6 +314,21 @@
                     $video = Html::tag('video', [$source], $attrs);
                 }
             } else {
+                // removes attributes that providers (youtube and vimeo) don't support in iframe
+                if (
+                    Str::contains($tag->value, 'youtu', true) === true ||
+                    Str::contains($tag->value, 'vimeo', true) === true
+                ) {
+                    unset(
+                        $attrs['autoplay'],
+                        $attrs['controls'],
+                        $attrs['loop'],
+                        $attrs['muted'],
+                        $attrs['poster'],
+                        $attrs['preload']
+                    );
+                }
+
```

Comment:
I had to remove attributes in video kirbytag that iframe doesn't support.

Reply:
This is probably exactly the reason why I added the allow list instead.

Reply:
I guess, yes 😅